pandas/io/pytables.py

```diff
@@ -3123,13 +3122,13 @@ def shape(self):
             node = self.group.block0_values
             shape = getattr(node, "shape", None)
             if shape is not None:
-                shape = list(shape[0 : (ndim - 1)])
+                shape_list = list(shape[0 : (ndim - 1)])
             else:
-                shape = []
+                shape_list = []
 
-            shape.append(items)
+            shape_list.append(items)
 
-            return shape
+            return tuple(shape)
```

Comment:
I made a mistake here and tests didn't fail. so will revert the changes to pandas/io/pytables.py for now.